lib/economic/endpoint.rb

```diff
@@ -43,12 +43,28 @@ def call(soap_action, data = nil, cookies = nil)
   # Cached on class-level to avoid loading the big WSDL file more than once (can
   # take several hundred megabytes of RAM after a while...)
   def client
-    @@client ||= Savon.client do
-      wsdl      File.expand_path(File.join(File.dirname(__FILE__), "economic.wsdl"))
-      log       false
-      log_level :info
-      headers("X-EconomicAppIdentifier" => @app_identifier) if @app_identifier
+    options = client_options
+    if @app_identifier
+      options = options.merge(
+        :headers => {
+          "X-EconomicAppIdentifier" => @app_identifier
+        }
+      )
```

Comment:
This could be simply

    options[:headers] = {"X-EconomicAppIdentifier" => @app_identifier}

which would probably silence Code Climate as well.